gwsurrogate/new/precessing_surrogate.py

```diff
@@ -764,21 +751,55 @@ def __call__(self, q, chiA, chiB, ellMax=4):
         modes = 1.j*np.zeros((nmodes, len(self.t)))
 
         for ell in range(2, ellMax+1):
-            # m=0 is different
-            re = _eval_comp(self.data['%s_0_real'%(ell)], q, chiA, chiB)
-            im = _eval_comp(self.data['%s_0_imag'%(ell)], q, chiA, chiB)
-            modes[ell*(ell+1) - 4] = re + 1.j*im
-
+            
+            # NOTE: modes is populated with zeros, so skipping means 
+            #       "set to zero". This is required for inertial_waveform_modes,
+            #       which maps from coorb to inertial frame modes. 
+            # m=0 has a different evaluation pattern
+            if (ell,0) in self.mode_list:
+                re = self._eval_comp(self.data['%s_0_real'%(ell)], q, chiA, chiB)
+                im = self._eval_comp(self.data['%s_0_imag'%(ell)], q, chiA, chiB)
+                modes[ell*(ell+1) - 4] = re + 1.j*im
+                #print("evaluation ell=%s, m=0"%ell)
+
+            # NOTE: modes is populated with zeros, so skipping means 
+            # "set to zero". This is required for inertial_waveform_modes,
+            # which maps from coorb to inertial frame modes. 
```

Comment:
Does this comment need to be repeated here? It's on line 755-757 above

Reply:
done!